packages/broker/test/integration/plugins/operator/VoteOnSusppectNodeService.test.ts

```diff
@@ -0,0 +1,131 @@
+import { Logger, waitForCondition } from '@streamr/utils'
+import { parseEther } from 'ethers/lib/utils'
+import StreamrClient from 'streamr-client'
+import { createWalletAndDeployOperator } from './deployOperatorContract'
+import { deploySponsorship } from './deploySponsorshipContract'
+import { Provider } from '@ethersproject/abstract-provider'
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { StreamrEnvDeployer, TestToken } from '@streamr/network-contracts'
+import { Wallet } from 'ethers'
+import { VoteOnSuspectNodeService } from '../../../../src/plugins/operator/VoteOnSuspectNodeService'
+import { MockProxy, mock } from 'jest-mock-extended'
+import { VoteOnSuspectNodeHelper } from '../../../../src/plugins/operator/VoteOnSuspectNodeHelper'
+import { Chain } from '@streamr/config'
+
+const theGraphUrl = `http://${process.env.STREAMR_DOCKER_DEV_HOST ?? '10.200.10.1'}:8000/subgraphs/name/streamr-dev/network-subgraphs`
+
+jest.setTimeout(600000)
```

Comment:
These are a bit iffy since they apply globally. Better to declare a TIMEOUT here and then pass it in for each `it` and `beforeEach` function. Also 10 minute timeout seems a bit long?

```suggestion
const TIMEOUT = 1000 * 60 * 10
```